### PR TITLE
restricting more, HOME and tmp in mpsyt.profile

### PR DESCRIPTION
**Pull request by pirate486743186**
Of course i got it wrong XD
Restricting HOME and tmp in mpsyt.profile

### CODE REVIEW

**Comment by glitsj16** [comment]
@pirate486743186 You might want to consider adding `ffmpeg` to `private-bin` for transcoding support. See $HOME/.config/mps-youtube/transcode for details.

File: etc/mpsyt.profile [hunk 1]
```diff
@@ -13,6 +13,18 @@ noblacklist ${HOME}/.netrc
 noblacklist ${HOME}/mps
 noblacklist ${MUSIC}
 noblacklist ${VIDEOS}
+noblacklist ${DOWNLOADS}
+
+mkdir ${HOME}/.config/mps-youtube
+
+whitelist ${HOME}/.config/mpv
```
Comment (SkewedZeppelin) [inline etc/mpsyt.profile]: these should be grouped with the other whitelists

File: etc/mpsyt.profile [hunk 2]
```diff
@@ -13,6 +13,18 @@ noblacklist ${HOME}/.netrc
...
+whitelist ${HOME}/.mplayer
+whitelist ${HOME}/.config/mps-youtube
+whitelist ${HOME}/.netrc
+whitelist ${HOME}/mps
+whitelist ${MUSIC}
+whitelist ${VIDEOS}
+whitelist ${DOWNLOADS}
```
Comment (SkewedZeppelin) [inline etc/mpsyt.profile]: you should also add the following above the other whitelist inc
`include /etc/firejail/whitelist-common.inc`

Comment (glitsj16) [reply etc/mpsyt.profile]: @SkewedZeppelin Shouldn't that be added without `/etc/firejail` as `include whitelist-common.inc`?

**Comment by SkewedZeppelin** [comment]
Merged, and fixed up in cc9db57343c3df0a9fc191e31273c955314c8f72

**Pull request merged**